commands/games/tictactoe.js

```diff
@@ -0,0 +1,170 @@
+import Canvas from 'canvas';
+import {Util, Attachment} from 'discord.js';
+import {CanvasUtils} from '../../utils/CanvasUtils';
+import {TicTacToe} from '../../games/TicTacToe';
+
+const games = {};
+const modes = ['easy', 'medium', 'hard'];
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
```js
const util = require('util');
const sleep = util.promisify(setTimeout);```

Reply:
but wouldn't promisify transform it in something like sleep(cb, ms)?
what would it return? or it would just make it the way I did?

Reply:
According to Node.js documentation it works fine like [this example](https://nodejs.org/api/timers.html#timers_settimeout_callback_delay_args).